src/android/Capture.java

```diff
@@ -288,11 +252,42 @@ private void captureAudio(Request req) {
         }
     }
 
+    /**
+     * Checks for and requests the camera permission if necessary.
+     *
+     * Returns a boolean which if true, signals that the permission has been granted, or that the
+     * permission isn't necessary and that the action may continue as normal.
+     *
+     * If the response is false, then the action should stop performing, as a permission prompt
+     * will be presented to the user. The action based on the request's requestCode will be invoked
+     * later.
+     *
+     * @param req
+     * @return
+     */
+    private boolean requestCameraPermission(Request req) {
+        boolean cameraPermissionGranted = true; // We will default to true, but if the manifest
+        // declares the permission, then we need to check
+        // for the grant
```

Comment:
```suggestion
        // Defaults to true. If the manifest declares permission then check for the grant
        boolean cameraPermissionGranted = true;
```

Could consolidate lines for comment.